ironfish/src/node.ts

```diff
@@ -124,11 +121,29 @@ export class IronfishNode {
       logger: logger,
     })
 
+    this.telemetry = new Telemetry({
+      chain,
+      logger,
+      config,
+      metrics,
+      workerPool,
+      localPeerIdentity: this.peerNetwork.localPeer.publicIdentity,
```

Comment:
Do you not want to pass the peer network in here like the other systems?

Reply:
Would it be possible in the future for `peerNetwork.localPeer.publicIdentity` to change? Maybe if a new identity is generated? If so yea maybe passing the whole peerNetwork would be better. I just didn't pass it to limit dependencies

Reply:
To me were copying some state from the concept it belongs to. I don't know why a telemetry has a localPeerIdentity, but if it was getting the state from the authority when it wanted to submit it, I would understand that instinctivly.